src/rust/cryptography-x509/src/name.rs

```diff
@@ -4,9 +4,11 @@
 
 use crate::common;
 
+pub type NameReadable<'a> = asn1::SequenceOf<'a, asn1::SetOf<'a, common::AttributeTypeValue<'a>>>;
```

Comment:
NB: I broke this out of the below because it's convenient to have as an alias in some of the validation APIs (e.g., APIs for retrieving subjects and issuers).

Reply:
FWIW, this would be trivial to merge if it was in its own PR.

Reply:
#9282